pkgs/development/python-modules/pytest-benchmark/default.nix

```
@@ -20,28 +29,52 @@ buildPythonPackage rec {
     hash = "sha256-f9Ty4+5PycraxoLUSa9JFusV5Cot6bBWKfOGHZIRR3o=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ionelmc/pytest-benchmark/commit/728752d2976ef53fde7e40beb3e55f09cf4d4736.patch";
+      hash = "sha256-WIQADCLey5Y79UJUj9J5E02HQ0O86xBh/3IeGLpVrWI=";
+    })
+  ];
+
   buildInputs = [
     pytest
   ];
 
   propagatedBuildInputs = [
     py-cpuinfo
-  ] ++ lib.optionals (pythonOlder "3.4") [
-    pathlib
-    statistics
   ];
 
-  # Circular dependency
-  doCheck = false;
-
   pythonImportsCheck = [
     "pytest_benchmark"
   ];
 
+  nativeCheckInputs = [
+    aspectlib
+    elasticsearch
+    freezegun
+    git
+    mercurial
+    pygal
+    pytestCheckHook
+  ];
+
+  preCheck = ''
+    export PATH="$out/bin:$PATH"
+  '';
+
+  disabledTests = [
+    # AttributeError: 'PluginImportFixer' object has no attribute 'find_spec'
+    "test_compare_1"
+    "test_compare_2"
+    "test_regression_checks"
+    "test_rendering"
+  ];
+
   meta = with lib; {
+    changelog = "https://github.com/ionelmc/pytest-benchmark/blob/${src.rev}/CHANGELOG.rst";
     description = "Pytest fixture for benchmarking code";
     homepage = "https://github.com/ionelmc/pytest-benchmark";
     license = licenses.bsd2;
-    maintainers = with maintainers; [ costrouc ];
+    maintainers = with maintainers; [ costrouc dotlambda ];
```

```suggestion
    maintainers = with maintainers; [ dotlambda ];
```
costrouc is MIA most of the time anyway